packages/suite/src/components/suite/NavigationBar/components/DeviceSelector/index.tsx

```diff
@@ -196,16 +196,18 @@ export const DeviceSelector = () => {
                             />
                         </DeviceLabel>
 
-                        <WalletNameWrapper>
-                            {selectedDevice.metadata.status === 'enabled' &&
-                            selectedDevice.metadata.walletLabel ? (
-                                selectedDevice.metadata.walletLabel
-                            ) : (
-                                <WalletLabeling device={selectedDevice} />
-                            )}
-
-                            <ArrowDown icon="ARROW_DOWN" size={16} />
-                        </WalletNameWrapper>
+                        {selectedDevice.state && (
+                            <WalletNameWrapper>
+                                {selectedDevice.metadata.status === 'enabled' &&
```

Comment:
nit: would be nice to destructure this `const {metadata} = selectedDevice `

Reply:
Since `selectedDevice` can be `undefined` I think it's cleaner to leave it as it is.